src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3871,9 +3878,24 @@ private VariableState GetVariableState(Optional<LocalState> localState)
             return (parameter, type, GetParameterAnnotations(parameter), isExpandedParamsArgument: false);
         }
 
-        private MethodSymbol InferMethodTypeArguments(BoundCall node, MethodSymbol method, ImmutableArray<BoundExpression> arguments)
+        private MethodSymbol InferMethodTypeArguments(BoundExpression node, MethodSymbol method, ImmutableArray<BoundExpression> arguments)
         {
-            return InferMethodTypeArguments(node.BinderOpt, method, arguments, node.ArgumentRefKindsOpt, node.ArgsToParamsOpt, node.Expanded);
+            return node switch
+            {
+                BoundCall call => InferMethodTypeArguments(call.BinderOpt,
+                                                           method,
+                                                           arguments,
+                                                           call.ArgumentRefKindsOpt,
+                                                           call.ArgsToParamsOpt,
+                                                           call.Expanded),
```

Comment:
It would be more consistent with the other complex call expressions in NullableWalker to change the formatting to something like the following.

```cs
BoundCall call => InferMethodTypeArguments(
    call.BinderOpt,
    method,
    arguments,
    call.ArgumentRefKindsOpt,
    call.ArgsToParamsOpt,
    call.Expanded)
```

I don't think this is important enough to hold up the PR, though.